templates/JSResolverOCHTTPS.js

```diff
@@ -1101,8 +1101,8 @@ function getReturnBlockFromSelection(selection, querySchemaInfo) {
 
 function resolveGrapgDBqueryForGraphQLMutation (queryAst, querySchemaInfo) {
```

Comment:
```suggestion
function resolveGraphDBqueryForGraphQLMutation (queryAst, querySchemaInfo) {
```

Misspelled function name